examples/detection/README.md

```diff
@@ -31,3 +31,11 @@ You can generate these visualization results by the following command.
 ```
 $ python visualuze_models.py
 ```
+
+## Notes on writing your own evaluation code
+
+Here are list of important configurations to reproduce results.
+
++ `model.use_preset('evaluate')` changes the score thresholds for NMS.
```

Comment:
`score_thresh` is not used in NMS. How about `configure postprocessing parameters for evaluation. (e.g. threshold of confidence score)`?